test/integration/TextEditorTest.kt

```diff
@@ -143,6 +144,10 @@ class TextEditorTest : IntegrationTest() {
                 clickText(composeRule, Label.SCHEMA.lowercase())
                 clickText(composeRule, Label.WRITE.lowercase())
 
+                waitUntilTrue(composeRule) {
+                    Service.driver.session.transaction.type == TypeDBTransaction.Type.WRITE
+                }
```

Comment:
A similar check is present in TypeBrowserTest.kt

Reply:
Hm- but something looks off here.

We open a schema session, verify that the session type is `SCHEMA`, and then click the button "Schema"? The button click seems redundant.

As a result we can just delete line 144 right?

Reply:
If anything I'd rather delete line 138 and rely on the clicks everywhere. (QueryRunnerTest e.g. has smth similar going on). @jamesreprise any idea on why we may need both?